projects/Mox/services/mox/module.nix

```diff
@@ -64,6 +64,7 @@ in
     ];
 
     networking.firewall = lib.mkIf cfg.openFirewall {
+      enable = true;
```

Comment:
This is bad practice.

https://github.com/NixOS/nixpkgs/blob/master/nixos/README.md?plain=1#L96-L97
> Ensure that the module respect other modules functionality.
> - For example, enabling a module should not open firewall ports by default.